bin/katello-certs-check

```diff
@@ -215,6 +215,29 @@ function check-cert-usage-key-encipherment () {
     fi
 }
 
+function check-shortname () {
+    printf "Checking for use of shortname as CN"
+
+    SUBJECT_CN=$(openssl x509 -in $CERT_FILE -noout -subject -nameopt multiline|grep commonName|cut -d '=' -f 2|tr -d ' ')
+    if [[ $SUBJECT_CN != *"."* ]]; then
+        error 1 "The $(basename $CERT_FILE) is using a shortname for Common Name (CN) and cannot be used with $PROJECT.\n"
+    fi
+
+    DNS_LIST=$(openssl x509 -noout -text -in $CERT_FILE | grep DNS:)
```

Comment:
`check-cert-san` ensures there is at least 1 SAN and it matches the subject. Since previously the subject was checked, we can safely assume that there is at least one SAN that is not a short name. That means this loop should be redundant. If it's not (and the test is hitting this) then is there a bug in `check-cert-san` or my logic here?

Reply:
I did consider that when building this. And I landed on doing this because this is a stand-alone check and has no expressed dependency on the ordering of the checks. If someone were to re-order them, we could lose out on a this check.

Reply:
Oh, there is also one potential flaw in my logic: `error` doesn't exit but rather it shows all the errors, right?

Reply:
Correct, `error` sets the error code and prints a message, but it does not exit immediately. So in theory, if you had 3 things wrong with your certificates, you would see all of them.